README.md

```diff
@@ -33,167 +29,162 @@ into any app.
 
-```java
-Alerter.isShowing();
+```kotlin
+Alerter.isShowing()
 ```
 
 To hide a currently showing Alert - 
 
-```java
-Alerter.hide();
+```kotlin
+Alerter.hide()
 ```
 
 # Customisation
 
 ### Background Colour
 
-```java
-Alerter.create(this)
+```kotlin
+Alerter.create(this@DemoActivity)
        .setTitle("Alert Title")
        .setText("Alert text...")
        .setBackgroundColorRes(R.color.colorAccent) // or setBackgroundColorInt(Color.CYAN)
-       .show();
+       .show()
 ```
 
 ![Coloured Alert](./documentation/alert_coloured.gif)
 
 ### Icon
 
-```java
-Alerter.create(this)
+```kotlin
+Alerter.create(this@DemoActivity)
        .setText("Alert text...")
        .setIcon(R.drawable.alerter_ic_mail_outline)
        .setIconColorFilter(0) // Optional - Removes white tint
-       .show();
+       .show()
 ```
 
 ![Custom Icon Alert](./documentation/alert_icon.gif)
 
 ### On screen duration, in milliseconds
 
-```java
-Alerter.create(this)
+```kotlin
+Alerter.create(this@DemoActivity)
        .setTitle("Alert Title")
        .setText("Alert text...")
        .setDuration(10000)
-       .show();
+       .show()
 ```
 
 ### Without title
 
-```java
-Alerter.create(this)
+```kotlin
+Alerter.create(this@DemoActivity)
        .setText("Alert text...")
-       .show();
+       .show()
 ```
 
 ![Text Only Alert](./documentation/alert_text_only.gif)
 
 ### Adding an On Click Listener
 
-```java
- Alerter.create(ExampleActivity.this)
+```kotlin
+ Alerter.create(this@DemoActivity)
         .setTitle("Alert Title")
         .setText("Alert text...")
         .setDuration(10000)
-        .setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                Toast.makeText(ExampleActivity.this, "OnClick Called", Toast.LENGTH_LONG).show();
-            }
+        .setOnClickListener(View.OnClickListener {
+            Toast.makeText(this@DemoActivity, "OnClick Called", Toast.LENGTH_LONG).show();
         })
-        .show();
+        .show()
 ```
 
 ![On Click Alert](./documentation/alert_on_click.gif)
 
 ### Verbose text
 
-```java
- Alerter.create(ExampleActivity.this)
+```kotlin
+ Alerter.create(this@DemoActivity)
         .setTitle("Alert Title")
         .setText("The alert scales to accommodate larger bodies of text. " +
                  "The alert scales to accommodate larger bodies of text. " +
                  "The alert scales to accommodate larger bodies of text.")
-        .show();
+        .show()
 ```
 
 ![Verbose Alert](./documentation/alert_verbose.gif)
 
 ### Visibility Callbacks
 
-```java
- Alerter.create(ExampleActivity.this)
+```kotlin
+ Alerter.create(this@DemoActivity)
         .setTitle("Alert Title")
-        .setOnShowListener(new OnShowAlertListener() {
-            @Override
-            public void onShow() {
-                Toast.makeText(ExampleActivity.this, "Alert Shown", Toast.LENGTH_LONG).show();
+        .setOnShowListener(object : OnShowAlertListener {
```

Comment:
Keep these interfaces as Java files and you can have the benefit of a lambda in Kotlin and keep the original Java caller the same. Like `View.OnClickListener {}`

Reply:
Oh handy, I'll revert them!